module/applications/actor/npc-sheet.mjs

```diff
@@ -180,8 +180,8 @@ export default class NPCActorSheet extends BaseActorSheet {
       secrets: this.actor.isOwner, relativeTo: this.actor, rollData: context.rollData
     };
     context.enriched = {
-      public: await TextEditor.enrichHTML(this.actor.system.details.biography.public, enrichmentOptions),
-      value: await TextEditor.enrichHTML(this.actor.system.details.biography.value, enrichmentOptions)
+      public: await CONFIG.ux.TextEditor.enrichHTML(this.actor.system.details.biography.public, enrichmentOptions),
+      value: await CONFIG.ux.TextEditor.enrichHTML(this.actor.system.details.biography.value, enrichmentOptions)
```

Comment:
We do have `const TextEditor = foundry.applications.ux.TextEditor.implementation;` at the top of this file, so we aren't calling the global here.